src/main/java/com/helloworld/hello/HelloController.java

```diff
@@ -3,10 +3,16 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.List;
+
 @RestController
 public class HelloController {
     @GetMapping("/helloworld")
```

Comment:
パスを
```suggestion
    @GetMapping("/countries")
```
のようにするとよいですね。

Reply:
パスを修正しました。
ご確認お願いいたします。

## コミットハッシュ
9dee6ac706e941ed71a784c5d03dcb3767972d0d
***
# 動作確認(POSTMAN)
<img width="1010" alt="スクリーンショット 2023-08-28 17 00 16" src="https://github.com/sugao-2211/task6/assets/141313076/4b77a295-aaac-4c1e-8cef-ac3f3d9baff4"><br>
***

# 動作確認(IntelliJ)
<img width="1005" alt="スクリーンショット 2023-08-28 16 56 31" src="https://github.com/sugao-2211/task6/assets/141313076/19354934-89fe-48f6-86a2-67c201ee3ebd">